### PR TITLE
hw8 done

### PR DESCRIPTION
**Pull request by anastasia1383**

### CODE REVIEW

File: hw_8/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+    const userBYear = prompt('Введіть Ваш рік народження.', '');
```
Comment (web-scavenger) [inline hw_8/index.js:2]: Try to avoid this magic `B`. it will be better to name like userBirthYear

File: hw_8/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+    const userBYear = prompt('Введіть Ваш рік народження.', '');  
+    const userCity = prompt('В якому місті Ви проживаєте?', '');
+    const userSport = prompt('Введіть Ваш улюблений вид спорту.', '');
+
+    const today = new Date().getFullYear();
+    const userAge = today - +userBYear;
```
Comment (web-scavenger) [inline hw_8/index.js:7]: It's okay but as far as you decided to convert to a number here -- you forgot about NaN. and `userAge`could be NaN

Comment (anastasia1383) [reply hw_8/index.js]: thank you for the comment, fixed this issue
can you check it, please

Comment (anastasia1383) [reply hw_8/index.js]: https://github.com/anastasia1383/VasylievaFrontEndPro251022/pull/6/commits/17fab78c46f0f440a627db78260b50d04f1f75b8

**Pull request merged**